tox.ini

```diff
@@ -70,7 +70,7 @@ ignore =
     sonar-project.properties
     tests/**
     tox.ini
-
+    config/rabbitmq/*
```

Comment:
Actually, you can't ignore these configuration files. You need to include them on the rhub-api package.
You can do that by including them under: [MANIFEST.in](https://github.com/resource-hub-dev/rhub-api/blob/master/MANIFEST.in)

Reply:
These configs are only for rabbitmq container, not for rhub-api.